### PR TITLE
BUGZ 887 Error in Webtablet.js

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/BUGZ-887

### CODE REVIEW

File: scripts/simplifiedUI/system/libraries/utils.js [hunk 1]
```diff
@@ -340,7 +340,7 @@ calculateHandSizeRatio = function() {
     return handSizeRatio;
 }
 
-clamp = function(val, min, max){
+hifiClamp = function (val, min, max) {
```
Comment (zfox23) [inline scripts/simplifiedUI/system/libraries/utils.js]:
```suggestion
hifiClamp = function(val, min, max) {
```

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20215/HighFidelity-Beta-PR15870-d840723.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20215/HighFidelity-Beta-Interface-PR15870-d840723.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20215/HighFidelity-Beta-PR15870-d840723.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20215/HighFidelity-Beta-Interface-PR15870-d840723.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20215/HighFidelity-Beta-PR15870-d840723.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20215/HighFidelity-Quest-Beta-PR15870-d840723.apk)

**Pull request merged**